src/twisted/web/test/test_flatten.py

```diff
@@ -9,7 +9,7 @@
 import sys
 import traceback
 
-from xml.etree.cElementTree import XML
```

Comment:
This part looks OK.

Reply:
This does look fine, and I wonder why it would provoke these Windows failures.